content/id/docs/reference/access-authn-authz/rbac.md

```diff
@@ -6,39 +6,40 @@ weight: 70
 ---
 
 <!-- overview -->
-Kontrol akses berbasis peran (RBAC) adalah metode pengaturan akses ke sumber daya komputer 
-atau jaringan berdasarkan peran pengguna individu dalam organisasi kamu.
+_Role-based access control_ (RBAC) atau kontrol akses berbasis rol adalah metode pengaturan akses ke sumber daya komputer 
+atau jaringan berdasarkan rol pengguna individu dalam organisasimu.
 
 
 <!-- body -->
-Otorisasi RBAC menggunakan `rbac.authorization.k8s.io` kelompok API untuk mengendalikan keputusan
-otorisasi, memungkinkan kamu untuk mengkonfigurasi kebijakan secara dinamis melalui API Kubernetes.
+Otorisasi RBAC menggunakan {{< glossary_tooltip text="grup API" term_id="api-group" >}} `rbac.authorization.k8s.io` untuk mengendalikan keputusan
+otorisasi. Hal ini memungkinkanmu untuk mengonfigurasi kebijakan secara dinamis melalui
+API Kubernetes.
 
-Untuk mengaktifkan RBAC, jalankan Kubernetes dengan _flag_ `--authorization-mode` atur 
-dengan daftar yang dipisahkan koma dengan menyertakan `RBAC`;
+Untuk mengaktifkan RBAC, jalankan {{< glossary_tooltip text="server API" term_id="kube-apiserver" >}} dengan _flag_ `--authorization-mode` diatur 
+dengan daftar yang dipisahkan koma yang menyertakan `RBAC`;
 sebagai contoh:
 ```shell
 kube-apiserver --authorization-mode=Example,RBAC --other-options --more-options
 ```
 
-## Objek API  {#api-overview}
+## Objek API {#api-overview}
 
 API RBAC mendeklarasikan empat jenis objek Kubernetes: Role, ClusterRole,
-RoleBinding and ClusterRoleBinding. kamu bisa [mendeskripsikan beberapa objek](/id/docs/concepts/overview/working-with-objects/kubernetes-objects/#understanding-kubernetes-objects), atau mengubahnya menggunakan alat seperti `kubectl`, seperti objek Kubernetes lain.
+RoleBinding dan ClusterRoleBinding. kamu bisa [mendeskripsikan beberapa objek](/id/docs/concepts/overview/working-with-objects/kubernetes-objects/#understanding-kubernetes-objects), atau mengubahnya menggunakan alat seperti `kubectl`, seperti objek Kubernetes lain.
 
 {{< caution >}}
 Objek-objek ini, dengan disengaja, memaksakan pembatasan akses. Jika kamu melakukan perubahan
 ke klaster saat kamu belajar, lihat
-[pencegahan eskalasi hak istimewa dan _bootstrap_](#privilege-eskalasi-pencegahan-dan-bootstrap)
+[pencegahan eskalasi privilese dan _bootstrapping_](#pencegahan-eskalasi-privilese-dan-bootstrapping)
 untuk memahami bagaimana pembatasan tersebut dapat mencegah kamu melakukan beberapa perubahan.
 {{< /caution >}}
 
 ### Role dan ClusterRole
 
-Sebuah RBAC Role atau ClusterRole berisi aturan yang mewakili sekumpulan izin.
+Sebuah Role RBAC atau ClusterRole memuat aturan yang mewakili sekumpulan izin.
 Izin bersifat aditif (tidak ada aturan "tolak").
 
-Sebuah Role selalu mengatur izin dalam Namespace tertentu;
+Sebuah Role selalu mengatur izin dalam {{< glossary_tooltip text="Namespace" term_id="namespace" >}} tertentu;
```

Comment:
we don't have namespace glossary localized in Bahasa Indonesia yet, feel free to add this glossary if you have time.